libs/remixd/src/bin/remixd.ts

```diff
@@ -83,7 +83,7 @@ function errorHandler (error: any, service: string) {
     console.log('\x1b[33m%s\x1b[0m', '[WARN] You may now only use IDE at ' + program.remixIde + ' to connect to that instance')
   }
 
-  if (program.sharedFolder) {
+  if (program.sharedFolder && existsSync(absolutePath('./', program.sharedFolder))) {
```

Comment:
`sharedFolder` can be either absolute path or relative path, is this test taking this in account?

Reply:
I have tested this and it seems fine to me